src/Hill.jl

```diff
@@ -134,3 +134,32 @@ function plotUnitSensitivity(paramRange, result, realParam, i)
     plot!([realParam], seriestype = "vline", margin = 0.3cm, legend = :false)
     ylims!((1E2, 1E4))
 end
+
+""" Calculate the # of cells in G1 for a set of parameters and T """
+function numG1(params, g0, T)
```

Comment:
Except you don't want numG1 or numG2, you want number of total cells. You should just have one function with these added together.